communities/sussex-hackers/community-assessment.md

```diff
@@ -0,0 +1,61 @@
+## Assessing Your Community's Needs
+
+Fully understanding the needs of your community is a complex concept that will likely take lots of time, conversation, and observation.  Depending on your situation, you may already have a great idea for what your community needs. You may not, and that's OK. We will go over these questions and topics broadly during week 1, and discuss responses in more detail in week 2. Here are some things to keep in mind when assessing your community and hoping to grow it.
+
+Please write down your
+- Answers to these questions for current community
+- Answers to these questions for **an ideal version** of your current community
+- Ideas on how to improve in each category
+
+After you've completed these questions, define your community in several short paragraphs for discussion in week 2.
+
+### Definition
+The current community I am a part of is hackers at Sussex University.
+Ideally this will grow to a community in and around Brighton (to connect the two universities in that area).
+  
+### Size
+The community is about 20 or so people, with low activity and low engagement.
+I would ideally like to see a larger number of students to get involved with more frequent events to foster engagement.
+The group is still less than a year old, probably 6 months and has been lacking in community leaders.
+Currently - I have no idea how to grow it.
```

Comment:
Reading your answers here, this isn't true! You know exactly how to grow it, you say frequently the things you'd like to see - these are things you think would make a difference. Have some confidence in yourself :)